### PR TITLE
RatPolls-08082022

### PR DESCRIPTION
**Pull request by Gala-G**
Ratification Polls for 08082022

### CODE REVIEW

File: governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP4c2-SP23 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP50 was too specific to lending protocols. I’ve updated it for the V2 rewrite which covers all external protocols that allow users to deposit / withdraw DAI. I’ve updated the MIP to remove details of the targeting mechanism as that is no longer important.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/4fc05f8a5056615397765d1daef2ed47e750c085)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md:36]: https://github.com/makerdao/mips/blob/4fc05f8a5056615397765d1daef2ed47e750c085/MIP4/MIP4c2-Subproposals/MIP4c2-SP23.md

File: governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* [Latest Proposal Version](https://forum.makerdao.com/t/mip4c2-sp23-mip50-direct-deposit-module-amendment/16387)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md:37]: https://mips.makerdao.com/mips/details/MIP4c2SP23

File: governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip4c2-sp23-mip50-direct-deposit-module-amendment/16387)
+
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md:39]: Add this:

```* [Amendment Pull Request](https://github.com/makerdao/mips/pull/606)```

File: governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, August 25.
+* Any further work required to implement the proposal will be tasked to the relevant [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md:46]: Maybe add this: "The Pull Request will be merged into the GitHub MIP repository" here? That's what I did [here](https://github.com/makerdao/community/pull/1900/files#diff-31a14075d7895b5c9114daff6f129b0a6d1a6d8aea5df380cc0fdd022b949280) ¯\_(ツ)_/¯

File: governance/polls/Ratification Poll for Makershire Redux (MIP76) - August 8, 2022.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP76 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Maker Protocol holds approximately $5.2B in stablecoins effectively earning zero yield. There are multiple efforts underway (off-chain and on-chain short-term bonds) to invest the majority of stablecoin exposure. Our aim is to begin generating yield on balance sheet assets and support DAI liquidity/utility, creating a new revenue stream for the protocol.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/e07030723333c8ce6259dfac63f0a47988cf17fb)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Makershire Redux (MIP76) - August 8, 2022.md:36]: https://github.com/makerdao/mips/blob/e07030723333c8ce6259dfac63f0a47988cf17fb/MIP76/MIP76.md

File: governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP4c2-SP23 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP50 was too specific to lending protocols. I’ve updated it for the V2 rewrite which covers all external protocols that allow users to deposit / withdraw DAI. I’ve updated the MIP to remove details of the targeting mechanism as that is no longer important.*
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md:33]: I suggest placing quotation marks around these summaries, in addition to italicizing

File: governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip4c2-sp23-mip50-direct-deposit-module-amendment/16387)
+* [Amendment Pull Request](https://github.com/makerdao/mips/pull/606)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, August 25.
+* The Pull Request will be merged into the GitHub MIP repository.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md:47]:
```suggestion
* The Pull Request will be merged into the GitHub MIPs repository.
```

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM, thanks

**Pull request merged**